src/creatwth.cpp

```diff
@@ -1304,8 +1306,12 @@ BOOL WINAPI DetourProcessViaHelperDllsW(_In_ DWORD dwTargetPid,
         goto Cleanup;
     }
 
-    hr = StringCchPrintfW(szCommand, ARRAYSIZE(szCommand),
-                          L"rundll32.exe \"%hs\",#1", &helper->rDlls[0]);
+	//for east asia languages and so on, like Chinese, print format with "%hs" can not work fine before user call _tsetlocale(LC_ALL,_T(".ACP"));
+    //so we can't use "%hs" in format string, because the dll that contain this code would inject to any process, even not call _tsetlocale(LC_ALL,_T(".ACP")) before
```

Comment:
nit pick: Can we fix the comment here and above to be consistent with the comment style with the rest of the code?

Reply:
indention of StringChhPrintf seems to be off by one, in both places you changed as well.

Reply:
The latest commit has resolved it.